index.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+
+
+app.get('/teams', (request, response) => {
+
+    response.send(teams)
+
+})
+
+app.get('/teams/', (request, response) => {
+
+    const allTeams = teams.map((team) => {
+
+        return team
+
+    })
+})
+
+
+
+app.get('/teams/:id', (request, response) => {
+    let matchingTeam = teams.filter((team) => {
+        return team.id === parseInt(request.params.id)
+    })
+    let matchingAbb = teams.filter((team) => {
+        return team.abbreviation === (request.params.id.toUpperCase())
+    })
+    if (!isNaN(request.params.id)) {
+        response.send(matchingTeam)
+    } else {
+        response.send(matchingAbb)
+    }
```

Comment:
Because you are always sending one of the two lists, if there is no matching teams, you will be sending a 200 with an empty list instead of a 404.